pkg/sh/exec.go

```diff
@@ -51,7 +51,7 @@ func RunOutput(ctx *task.Context, name string, args ...string) (string, error) {
 // RunCmd runs the provided command.
 func RunCmd(ctx *task.Context, cmd *exec.Cmd) error {
 	LogCmd(ctx, cmd)
-	if ctx.Verbose && cmd.Stdout == nil {
+	if cmd.Stdout == nil {
```

Comment:
I am not sure if you actually want this change. Perhaps you would prefer the usage to require `-v` for test output.

Reply:
yeah, the intent here is that -v is required if you want to see the content.